docs/background-information/publishers.md

```diff
@@ -0,0 +1,92 @@
+# What publisher is right for you?
+
+Wagtail Live supports various publishers. 
+Their task is to update the live page every time a post is created.
+
+
+## Interval polling 
+
+[Interval polling](/getting_started/publishers/setup_interval_polling.md) is Javascript running in the browser, that asks the webserver for updates with a regular interval.
```

Comment:
What is the default interval? Let's add it here, so readers understand if it's typically seconds or minutes.